nilearn/decoding/space_net.py

```diff
@@ -458,8 +458,11 @@ def path_scores(solver, X, y, mask, alphas, l1_ratios, train, test,
             all_test_scores.append(this_test_scores)
     else:
         if alphas is None:
-            raise RuntimeError
-        alphas_ = alphas
+            alphas_ = _space_net_alpha_grid(
```

Comment:
Not sure why you did this in this PR, but you changed the meaning of `alphas=None` can you add a test for this as well?